### PR TITLE
Button: Mobile Align Setting

### PR DESCRIPTION
**Pull request by AlexGStapleton**
This PR adds a Mobile Align setting to the SO Button Widget. This allows users to change the alignment of the button on mobile to better suit their design. This PR also includes a Responsive Global Addon setting to allow the user to control when the mobile align setting kicks in.

### CODE REVIEW

File: widgets/button/styles/atom.less [hunk 1]
```diff
@@ -18,6 +21,23 @@
...
+
+		& when not ( @mobile_align = justify ) {
+			&.ow-button-align-justify a {
+				display: inline-block;
+			}
+		}
+
+		& when ( @mobile_align = justify ) {
+			a {
+				dispaly: block;
```
Comment (Misplon) [inline widgets/button/styles/atom.less:35]: Typo here.

File: widgets/button/styles/wire.less [hunk 2]
```diff
@@ -18,6 +21,23 @@
...
+
+		& when not ( @mobile_align = justify ) {
+			&.ow-button-align-justify a {
+				display: inline-block;
+			}
+		}
+
+		& when ( @mobile_align = justify ) {
+			a {
+				dispaly: block;
```
Comment (Misplon) [inline widgets/button/styles/wire.less:35]: Typo here.

**Comment by Misplon** [comment]
@AlexGStapleton Let's change `Mobile Align` to `Mobile align` to match the formatting of our other settings. Thanks, mate.

**Comment by Misplon** [comment]
The new rules are being overwritten due to specificity.

DevTools_-_localhost_siteorigin_page-builder_

**Comment by AlexGStapleton** [comment]
Oddly, that's not happening for me in Firefox or Chrome. Regardless, the last commit is much more specific so the issue shouldn't occur anymore.

**Review by Misplon — APPROVED** [review]

**Review by gregpriday — APPROVED** [review]

**Pull request merged**